src/main/java/seedu/address/model/group/Group.java

```diff
@@ -1,31 +1,15 @@
 package seedu.address.model.group;
 
-import static java.util.Objects.requireNonNull;
-
 /**
  * Represents a contact group in the address book.
  */
 public class Group {
 
     private String groupName;
 
```

Comment:
I think our Group class should also contain a list of person in the group

Reply:
After the person are assigned to a group, the group will contain a list of person and be saved in the addressbook.json file.

Reply:
Will it be added in? Cause currently, the addressbook.json only store the list of group created, it didn't store who is inside the group.